plugins/source/aws/codegen/recipes/base.go

```diff
@@ -39,6 +39,9 @@ type Resource struct {
 	// --------------------------------
 	ShouldGenerateResolverAndMockTest bool
 	ResolverAndMockTestTemplate       string
+
+	// To be used in list/paginator resolver
+	MaxResults int
```

Comment:
Do we want to split the codegen changes into a separate PR?

Reply:
I specifically added the new templates to gen the new resources, so maybe leave in single PR?
If you feel strongly it should be split I can do it, though.